src/graph/test/DataTest.cpp

```diff
@@ -359,6 +359,23 @@ TEST_F(DataTest, InsertVertex) {
     // TODO: Test insert multi tags, and delete one of them then check other existent
 }
 
+TEST_F(DataTest, FindTest) {
+    {
```

Comment:
why don't support some `SUCCEEDED` example ?

Reply:
Because they have not to be implemented